doc/source/_toc.yml

```diff
@@ -11,6 +11,12 @@ parts:
         title: "Installation"
       - file: ray-overview/use-cases
         title: "Use Cases"
+        sections:
+          - file: ray-air/getting-started
+            sections:
+              - file: ray-air/deployment
```

Comment:
Suggest moving this content into Ray for ML Infrastructure to reduce the number of levels.